TheJumpCloud/jcapi-powershell
# exclude Application from V2 endpoint

## Issues
* [SA-2640](https://jumpcloud.atlassian.net/browse/SA-2640) - Remove duplicate function Get-JcSdkApplication from V2 Module

## What does this solve?

In the previous release, a duplicate function was included in the V2 module. This release excludes building this function. Downstream this created an issue for Backup-JCOrganization. 

## Is there anything particularly tricky?

No, only a single function Get-JCSdkApplication was removed, only other changes are model names and descriptions.

## How should this be tested?

Pull this repository and this branch. Change directories into the /jcapi-powershell repository
Run `./BuildAutoRest -SDKName JumpCloud.Sdk.V2` the V2 module should build.
Import the newly generated module `Import-Module "./SDKs/PowerShell/JumpCloud.SDK.V2/JumpCloud.SDK.V2.psd1" -force`
Verify that the function `Get-JcSdkApplication` does not exist in the V2 Module. Running `Get-JcSdkApplication` function should return all applications in your org (this is the V1 function)

Also verify that the generated JumpCloud.SDK.V2.psd1 file does not include the `Get-JcSdkApplication` function in the `FunctionsToExport` table

## Screenshots

Get-JCSdkApplication has been removed:
![Screen Shot 2022-06-20 at 1 21 50 PM](https://user-images.githubusercontent.com/54448601/174666280-16fa9e6b-c46c-4b73-a96a-74ff855945e6.png)
